packages/docs/content/patterns/utility-actions.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Utility Actions
+intro: Utility actions for a webpage including print, email, RSS and more.
+---
+
+import {
+  DownloadIcon,
+  EmailIcon,
+  LinkIcon,
+  PrintIcon,
+  RssIcon,
+  WhatsNewIcon,
+} from '@cmsgov/design-system';
+
+## Guidance
+
+### Examples
+
+<EmbeddedExample>
+  <div className="ds-l-row">
+    <div className="ds-l-col--2">
+      <PrintIcon viewBox="0 0 36 36" />
+      <p className="ds-u-margin--0">Print</p>
+    </div>
+    <div className="ds-l-col--2">
+      <EmailIcon viewBox="0 0 36 36" />
+      <p className="ds-u-margin--0">Email</p>
+    </div>
+    <div className="ds-l-col--2">
+      <RssIcon viewBox="0 0 36 36" />
+      <p className="ds-u-margin--0">RSS</p>
+    </div>
+    <div className="ds-l-col--2">
+      <LinkIcon viewBox="0 0 36 36" />
+      <p className="ds-u-margin--0">Link</p>
+    </div>
+    <div className="ds-l-col--2">
+      <DownloadIcon viewBox="0 0 36 36" />
+      <p className="ds-u-margin--0">Download</p>
+    </div>
+    <div className="ds-l-col--2">
+      <WhatsNewIcon viewBox="0 0 36 36" />
+      <p className="ds-u-margin--0">What's New</p>
+    </div>
+  </div>
+</EmbeddedExample>
+
+### When to use
+
+- If you would like to give the user the option to print or email a page.
+- If you would like to give the user the option to enable RSS.
+- These "utility actions" are considered [ghost buttons](/components/button/#ghost-button) for actions available that do not take priority over a primary or secondary CTAs (e.g., print, email, RSS).
```

Comment:
CTAs should be reworded - not everyone understands that abbreviation

Reply:
I wonder if RSS needs an explainer somewhere as well? Devs might know what that means, but other folks may not